### ISSUE TITLE
segfault on empty workspace

### ISSUE THREAD

**Issue by darthoctopus** [rilian-la-te/vala-panel-appmenu#36]
vala-panel and mate-panel segfault on empty workspaces when vala-panel-appmenu is enabled (because of this, vala-panel cannot be used as MATE required-component panel option)

**Comment by rilian-la-te** [comment]
Yes, it is true:) CSD Apps segfaults also because Compiz try to switch to empty workspace first:)

**Comment by rilian-la-te** [comment]
Seems fixed.
